app/bin/service/analyzer.dart

```diff
@@ -30,14 +30,16 @@ Future main() async {
 
   withAppEngineServices(() async {
     _initFlutterSdk().then((_) async {
-      startIsolates(logger, _runScheduler);
+      startIsolates(logger, _runSchedulerWrapper);
     });
     _registerServices();
     await runAppEngine((HttpRequest request) =>
         shelf_io.handleRequest(request, analyzerServiceHandler));
   });
 }
 
+void _runSchedulerWrapper(message) => _runScheduler(message);
```

Comment:
Put a comment about this referencing https://github.com/dart-lang/sdk/issues/30755